eultheme/tests.py

```diff
@@ -1,3 +1,77 @@
+import datetime
+
 from django.test import TestCase
+from .models import Banner, DowntimePeriod
+
+class BannerTest(TestCase):
+    def test_is_active(self):
+        """
+        Tests the banner will be visible when the date range is applicable
+        """
+        self.p1 = DowntimePeriod.objects.create(
+            id=123,
+            enabled=True,
+            start_time = datetime.datetime.now(),
+            end_time = datetime.datetime.now() + datetime.timedelta(days=2)
+        )
+        self.b1 = Banner.objects.create(
+            id=1234,
+            message = "Test banner",
+            period = self.p1,
+            days=0
+        )
+
+        self.active_banner = Banner.objects.get_deployed().first()
```

Comment:
It's hard for me to tell if you're hitting all the cases here.  Might be worth adding comments and doing them systematically? past/present/future with combinations of active and no end?